DemoUI/index.html

```diff
@@ -187,12 +194,43 @@
           <i class='fa fa-angle-left backbutton' id='backup-detail-done-button'></i>
         </menu>
         <h1 id='detail-page-title'>EMPTY !!</h1>
-      </header>
+
+</header>
+
+
+      
 
       <section class='gaia-list fit scroll sticky' id='main-page' style='margin-top: 5rem; position: absolute'>
         <div data-type='list' style="overflow-y: scroll; height: calc(100% - 10rem); padding: 0">
-        </div>
+  
+
+  
+    
+  <table>
+    <tr>
+      <td style="width: 25%" id='detail-page-title-2'>EMPTY</td>
```

Comment:
We should consider consolidating these into a single element (likely a paragraph) so that data types with differing lengths (e.g. "Photos" vs. "Contacts") don't have variable amounts of whitespace.